libs/amp-evaluation/samples/06-decorator-evaluator/README.md

```
@@ -0,0 +1,65 @@
+Both approaches produce `BaseEvaluator` instances that are fully interchangeable in `Monitor`, `Experiment`, and `discover_evaluators()`.
+
+## How to run
+
+```bash
+pip install amp-evaluation
+python run.py
+```
+
+## Expected output
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)